portfolio/src/main/webapp/style.css

```diff
@@ -19,4 +19,8 @@
 
 .title:hover {
   background: #ddd;
+}
+
+.collapsed {
```

Comment:
I was thinking of instead putting the `.collapsed` class on the same element that already has the `.section` class and then having a rule like:
```
.section.collapsed > .description {
  display: none;
}
```

See https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator
This also makes the classes human-readable because you see `section` and `collapsed` on the same element. And it will simplify you JS logic even more so you only need to access one DOM element by id.

Reply:
Great idea!